griptape/drivers/vector/mongodb_vector_store_driver.py

```diff
@@ -98,6 +98,9 @@ def query(
         pipeline = [
```

Comment:
I think those are old changes. Can you remove them from the PR?

Reply:
This was just me updating the branch, sorry!

Reply:
I can still see those changes.